julia.html.markdown

```diff
@@ -744,14 +744,14 @@ fight(Lion("RAR"), Lion("brown", "rarrr"))  # => The lions come to a tie
 # Under the hood
 # You can take a look at the llvm  and the assembly code generated.
 
-square_area(l) = l * l  # square_area (generic function with 1 method)
+squareArea(l) = l * l  # squareArea (generic function with 1 method)
 
-square_area(5)  # => 25
+squareArea(5)  # => 25
 
-# What happens when we feed square_area an integer?
-code_native(square_area, (Int32,), syntax = :intel)
+# What happens when we feed squareArea an integer?
+codeNative(squareArea, (Int32,), syntax = :intel)
 	#         .text
-	# ; Function square_area {
+	# ; Function squareArea {
```

Comment:
Function name needs to be changed. Refer https://github.com/adambard/learnxinyminutes-docs/pull/3256#issuecomment-427193194